ide/languages.hcl/src/org/netbeans/modules/languages/hcl/terraform/TerraformParserResult.java

```diff
@@ -116,8 +122,11 @@ protected void processDocument(HCLDocument doc, SourceRef references) {
             } else {
                 references.getOffsetRange(type).ifPresent((range) -> addError(Bundle.UNKNOWN_BLOCK(type.id()), range));
             }
+            checkDuplicateAttribute(doc, references);
```

Comment:
This leads to errors reported multiple times. This either needs to be moved out of the loop (started in new line 101) or doc must become `block`:

```suggestion
            checkDuplicateAttribute(block, references);
```

Reply:
Valid.